ocean_lib/exceptions.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+class ContractNotFound(Exception):
```

Comment:
For this exception class, and all the other exception classes in this file:
-The docstring should all be on one line, including triple-quotes
-The class definition needs a "pass" (I'm surprised that unit tests passed without having this, hmmm..)

Example of correct way:
----
class ContractNotFound(Exception):
    """Contract address is not found in the factory events"""
    pass
----
More examples:
https://www.programiz.com/python-programming/user-defined-exception

Reply:
This `pass` make the codacy report fails.
https://pycodequ.al/docs/pylint-messages/w0107-unnecessary-pass.html